recipes/libsimpleitk/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: libsimpleitk
+  version: "{{version}}"
+
+source:
+  git_url: git://itk.org/SimpleITK.git
+  git_tag:  v{{version}}
+  patches:
+    - fixes.patch
```

Comment:
Some description of the patch in a comment in the `meta.yaml` would be good. Can put the comment above the patch. Also would be nice to see a link to a PR or issue where the problem has been raised with upstream and these fixes included.